files/sanity_check.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+import socket
+import sys
+import json
+import urllib2
+
+
+REPORT = []
+
+
+def check_port(host, port):
+    try:
+        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        s.settimeout(.1)
+        s.connect((host, port))
+        s.shutdown(socket.SHUT_RDWR)
+        return True
+    except socket.error:
+        return False
+
+
+def check_command(command):
+    try:
+        devnull = open(os.devnull, 'w')
+        subprocess.check_call(
+            command, shell=True, stdout=devnull, stderr=subprocess.STDOUT
+        )
+        return True
+    except subprocess.CalledProcessError:
+        return False
+
+
+def get_free_percentage_on_mount(mount):
+    stats = os.statvfs(mount)
+    return stats.f_bavail / float(stats.f_blocks) * 100
+
+
+def test_free_space_on_mount(mount, free_percentage_required):
+    free_percentage = get_free_percentage_on_mount(mount)
+    if free_percentage_required > free_percentage:
+        REPORT.append("Not enough free space on {mount}. Free space is {percentage:.1f}%".format(
+            mount=mount,
+            percentage=free_percentage
+        ))
+
+
+def send_message(recipient, subject, body):
+    try:
+      process = subprocess.Popen(['mail', '-s', subject, recipient], stdin=subprocess.PIPE)
+      process.communicate(body)
+    except Exception as  error:
+        print(error)
```

Comment:
No real reason (apart from me being tired :(). In the long run it has no consequence whatsoever, Becouse mail doesn't really catch much errors, I moved send_message to the last step of sanity check, so this is after snitch, and output from this command is ignored in cron. 

Nevertheless this is now fixed.